src/zcl_aff_tools_unit_test_helper.clas.abap

```diff
@@ -0,0 +1,138 @@
+CLASS zcl_aff_tools_unit_test_helper DEFINITION FINAL FOR TESTING
```

Comment:
I personally add a `T` to the class name to indicate that this class is a test class and cannot be called from production system. So this would be `ztcl_aff_tools_unit_test_helper` (similar to local test classes starting with `ltlc`

@all Any opinion on this?

Reply:
I agree with you. I will add a `T` to the classname

Reply:
Adding a `T` results in a length that exceeds the maximum length of 30 characters and we would have to change the whole name. Is it worth it?

Reply:
No, I think it's not worth. Thanks for checking.